perf/src/main/java/org/bf2/performance/InstanceProfiler.java

```diff
@@ -311,8 +317,8 @@ public static void main(String[] args) throws Exception {
     private void runLocalTest() throws Exception {
         Profile profile = LATENCY;
         sizeAndUpdateConfig();
-        deployIfNeeded("profile");
-        OMBDriver driver = createDriver(instanceBootstrap, profile);
+        deployIfNeeded();
+        updateDriver(profile);
 
         byte[] storeBytes = Base64.getDecoder().decode(kafkaProvisioner.getTlsConfig().getTrustStoreBase64());
         File store = new File("target", "listener.jks");
```

Comment:
It doesn't need to be fixed here, but when I used this method locally, I needed to replace the keystore load with first reading `ssl.truststore.location` from the `appServicesProperties`.

Reply:
Yes, I haven't ever run the local test execution against a kas-installer cluster.  This method is mostly for debugging purposes, so I'm fine with whatever changes here.